packages/cli/config/eslint-factory.js

```diff
@@ -44,6 +45,7 @@ function createConfig(dir, extraConfig = {}) {
     testRules,
 
     restrictedImports,
+    restrictedImportsPatterns,
```

Comment:
I think naming-wise it probably is more natural to use `restrictedImportPatterns` right? Thinking especially with potential future `restrictedImportSrcPatterns` etc.